tft.py

```diff
@@ -109,8 +109,11 @@ def kill_process(process_executable: str, force: bool = True) -> subprocess.Comp
     Returns:
         str: _description_
     """
+    task_kill_args = ["taskkill", "/im", process_executable]
+    if force:
+        task_kill_args.insert(1, "/f")

```

Comment:
Oh I thought you were sticking to the string? 😛 
Either of these approaches work for me 👍

Reply:
Yeah, why not keep it as-is? The one benefit we get from this is that we can pass an argument with spaces, and it'll consider it one argument. If you check the string I had before, I had to add escaped quotes for this explicitly.

Reply:
Very true -- sounds good!